index.html

```diff
@@ -47,5 +51,132 @@
+            <h3>Health DNA test + Wellness Plans</h3>
+            <h1>
+              Get The Only
+              <a
+                href="https://zpe.gov.pl/a/dna---nosnik-informacji-genetycznej/D18yvChME"
+                >DNA</a
+              >
+              Test That Provides Truly Personalized Insights Based on Your
+              Genetic Results
+            </h1>
+          </div>
+
+          <div class="hero_frame">
+            <div class="frame_text">
+              <i class="fa-solid fa-square-check"></i>
+              <p>Analysis of health-related genetic variants</p>
+            </div>
+            <div class="frame_text">
+              <i class="fa-solid fa-square-check"></i>
+              <p>Summary of genetic risks</p>
+            </div>
+            <div class="frame_text">
+              <i class="fa-solid fa-square-check"></i>
+              <p>40+ personalized wellness reports</p>
+            </div>
+            <div class="frame_text">
+              <i class="fa-solid fa-square-check"></i>
+              <p>Custom health pln builder</p>
+            </div>
+          </div>
+          <div class="hero_button-price">
+            <button class="submit_button">
+              order now<i class="fa-solid fa-circle-chevron-right"></i>
+            </button>
+            <div class="price_block">
+              <h1>Only For</h1>
+              <div class="money_block">
+                <p>$199</p>
+                <p><del>$398</del></p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="hero_img">
+          <img src="./public/images/heroIMG.png" alt="hero_img" />
+        </div>
+      </section>
+
+      <section id="dna_info">
+        <div class="ifo_text">
+          <h1>What is DNA?</h1>
+          <p>
+            DNA (deoxyribonucleic acid) contains your genetic code, and it's
+            unique. You get a copy of your genes from each of your parents. The
+            places where your genes are different are called genetic variants.
+          </p>
+          <p>
+            Sometimes you end up with a variation in your genetic code that is
+            associated with a health issue. Fortunately, you can change the
+            expression of many of your genes.
+          </p>
+        </div>
+        <div class="movie">
+          <video controls width="250">
+            <source src="./public/movies/DNAVideo.mp4" type="video/mp4" />
+          </video>
+        </div>
+      </section>
+
+      <section id="analyze">
+        <h1>You spit. We analyze.</h1>
+        <p>
+          Your DNA influences your health, so what does that mean for you? Find
+          out with personalized reports covering topics like:
+        </p>
+        <div class="img_analyze">
+          <img src="./public/images/6.png" alt="img" />
+        </div>
+        <div class="human_analyze">
+          <div class="blood">
+            <div class="blood_text">
```

Comment:
blood text but you have image inside this div... where is a logic of this naming?